pdf_rendering.py

```diff
@@ -1,239 +1,378 @@
 import fpdf
 import copy
+#TODO OVERALL: MODIFY TEXT OUTPUT AS MORE INFORMATION BECOMES AVAILABLE IN INVOICE
+
+
 def render_activity_report(invoice):
     pdf = fpdf.FPDF(orientation='P', unit='mm', format='A4')
     pdf.add_page()
-    pdf.set_font("Arial")
+    pdf.set_font('Arial')
```

Comment:
Let's use Helvetica since it looks so much better, on windows machines it falls back to Arial anyway.